lms/djangoapps/api_manager/courses/tests.py

```diff
@@ -1615,10 +1609,10 @@ def test_courses_metrics_social_get(self):
         test_uri = '{}/{}/metrics/social/'.format(self.base_courses_uri, self.test_course_id)
         response = self.do_get(test_uri)
         self.assertEqual(response.status_code, 200)
-        self.assertTrue(len(response.data.keys()), 2)
+        self.assertEqual(len(response.data.keys()), 2)
         users = response.data['users']
-        self.assertTrue(users.get('1'))
         self.assertTrue(users.get('2'))
+        self.assertTrue(users.get('3'))
```

Comment:
How about

    self.assertIn('2', users)

This tests something slightly different, since it won't check that the value evaluate to `True`, but my  feeling is that is tests what we actually want to test. :)